trillian/ctfe/configpb/config.proto

```diff
@@ -101,6 +101,16 @@ message LogConfig {
   // log's key and can't sign STHs. Consequently, the log operator must ensure
   // to channel source log's STHs into CTFE.
   bool is_mirror = 12;
+
+  // The Maximum Merge Delay (MMD) of this log in seconds. See RFC6962 section 3
```

Comment:
That might be > 80 chars or it be github display. Also below.

Reply:
Checked, it's exactly 80.